cephfs/file_ops.go

```diff
@@ -32,3 +32,29 @@ func (mount *MountInfo) Mknod(path string, mode uint16, dev uint16) error {
 	ret := C.ceph_mknod(mount.mount, cPath, C.mode_t(mode), C.dev_t(dev))
 	return getError(ret)
 }
+
+// Utime struct is the equivalent of C.struct_utimbuf
+type Utime struct {
+	AcTime  int64
+	ModTime int64
```

Comment:
Optional: Since these abbreviated names match the C struct they're a bit terse and may not be so obvious. Can we add short doc comments?
```suggestion
	// AcTime  represents the file's access time in seconds since the Unix epoch.
	AcTime  int64
	// ModTime represents the files' modification time in seconds since the Unix epoch.
	ModTime int64
```

Reply:
Makes sense. I have updated the PR to add these doc comments.